charts/incubator/miniflux/values.yaml

```diff
@@ -28,24 +15,41 @@ persistence:
     enabled: true
     mountPath: "/config"
 
-probes:
-  liveness:
+cnpg:
+  main:
     enabled: true
-    custom: true
-    spec:
-      initialDelaySeconds: 30
-      periodSeconds: 10
-      timeoutSeconds: 1
-      failureThreshold: 3
-      httpGet:
-        path: /healthcheck
-        port: 8080
-
-postgresql:
-  enabled: true
-  existingSecret: "dbcreds"
-  postgresqlDatabase: miniflux
-  postgresqlUsername: miniflux
+    user: miniflux
+    database: miniflux
 
 portal:
-  enabled: true
+  open:
+    enabled: true
+
+securityContext:
+  container:
+    runAsUser: 568
+    runAsGroup: 568
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              enabled: true
+              type: http
+              path: /healthcheck
+            readiness:
+              enabled: true
+              type: http
+              path: /healthcheck
+          env:
+            RUN_MIGRATIONS: 1
+            CREATE_ADMIN: 1
```

Comment:
Is there a reason to **not** run migrations or **not** create an admin?
If not lets remove it from the questions.yaml

Reply:
For a fully managed application from TrueNAS SCALE UI, there is no reason indeed, I'll remove those from the questions.yaml.

Reply:
updated in 3a89517d49908b4c067a466f4114c6e6a1eaa41b.